code/modules/cargo/packs.dm

```diff
@@ -909,7 +909,7 @@
 
 /datum/supply_pack/science/bz_canister
 	name = "BZ Canister"
-	cost = 4000
+	cost = 2000
```

Comment:
No. Bad Jughu. BAD Jughu. BZ does not need to be cheaper. It needs to be more expensive.